project/Build.scala

```diff
@@ -81,7 +81,7 @@ object Build {
    *  set to 3.1.3. If it is going to be 3.1.0, it must be set to the latest
    *  3.0.x release.
    */
-  val previousDottyVersion = "3.0.0"
+  val previousDottyVersion = "3.0.1-RC1"
```

Comment:
We must never use an RC here anymore. This should stay 3.0.0. It should become 3.0.1 when that is released. The reason is that, during an RC cycle, we might still decide to remove one of the additions (because it would have badly broken someone's code, for example).